awx/main/utils/plugins.py

```diff
@@ -56,4 +56,31 @@ def load_combined_inventory_source_options() -> dict[str, str]:
 
     plugins = compute_cloud_inventory_sources()
 
-    return dict(zip(plugins, plugins), file='file')
+    plugin_descriptions = {
+        'azure_rm': 'Microsoft Azure Resource Manager',
```

Comment:
The value here should live next to the plugins themselves over in `awx-plugins`

Reply:
That makes 100% sense! I was missing a few process steps here that I didn't understand. I now see how this will be consumed.